camera/android/src/main/java/com/capacitorjs/plugins/camera/CameraBottomSheetDialogFragment.java

```diff
@@ -113,10 +113,11 @@ public void setupDialog(Dialog dialog, int style) {
         dialog.setContentView(parentLayout.getRootView());
 
         CoordinatorLayout.LayoutParams params = (CoordinatorLayout.LayoutParams) ((View) parentLayout.getParent()).getLayoutParams();
-        CoordinatorLayout.Behavior behavior = params.getBehavior();
+        CoordinatorLayout.Behavior<?> behavior = params.getBehavior();
 
-        if (behavior != null && behavior instanceof BottomSheetBehavior) {
-            ((BottomSheetBehavior) behavior).addBottomSheetCallback(mBottomSheetBehaviorCallback);
+        if (behavior instanceof BottomSheetBehavior) {
+            ((BottomSheetBehavior<?>) behavior).addBottomSheetCallback(mBottomSheetBehaviorCallback);
+            ((BottomSheetBehavior<?>) behavior).setState(BottomSheetBehavior.STATE_EXPANDED);
```

Comment:
Couldn't we use `BottomSheetBehavior<View>` here? 
Also, wouldn't it be better to create a local variable and then use it for both calls? This way we could avoid having `((BottomSheetBehavior<?>) behavior)` duplicated.